src/Microsoft.Framework.FileSystemGlobbing/Internal/PathSegments/LiteralPathSegment.cs

```diff
@@ -29,7 +33,7 @@ public override bool Equals(object obj)
             var other = obj as LiteralPathSegment;
 
             return other != null &&
-                string.Equals(other.Value, Value, StringComparison.Ordinal);
+                string.Equals(other.Value, Value, _comparisonType);
```

Comment:
You're right. I'm wondering if it is even necessary to override the Equals here.